Datasets/LanguageModelDataset.swift

```diff
@@ -105,28 +99,12 @@ extension LanguageModelDataset: Collection {
   }
 }
 
-//Extension when Item is [Int] and openItem is not needed
-extension LanguageModelDataset where Item == [Int] {
-  public init(batchSize: Int, sequenceLength: Int, items: [Item], lengths: [Int]) {
-    self.init(
-      openItem: { $0 },
-      batchSize: batchSize,
-      sequenceLength: sequenceLength,
-      items: items,
-      lengths: lengths)
-  }
-
-  public init(batchSize: Int, sequenceLength: Int, items: [Item]) {
-    self.init(
-      openItem: { $0 },
-      batchSize: batchSize,
-      sequenceLength: sequenceLength,
-      items: items,
-      lengths: items.map { $0.count })
-  }
-}
-
-//sampleIndices function to use in conjunction with a LanguageModelDataset
+/// The sampleIndices function to use in conjunction with a `LanguageModelDataset` in a `Batcher`.
+/// Will shuffle the dataset in place instead of the indices (like the default function does).
+/// - Parameters:
+///   - dataset: The underlying `LanguageModelDataset`
```

Comment:
Please consistently use periods.
```suggestion
///   - dataset: The underlying `LanguageModelDataset`.
```